### PR TITLE
Ft/font normalization

### PR DESCRIPTION
**Pull request by kelvinkipruto**
This PR ensures similar font sizes are used on `welcome` and `opportunities` components

font-33
font3
font2
font1

### CODE REVIEW

File: src/app/opportunities-early-adopters/opportunities-early-adopters-page.component.html [hunk 1]
```diff
@@ -62,7 +62,7 @@ <h1>Call for Applications for Early Adopters to Combat Online Gender-Based Haras
         effectiveness to their followers and networks.
       </p>
 
-      <h4>Who can apply?</h4>
+      <h4 class="title">Who can apply?</h4>
```
Comment (kilemensi) [inline src/app/opportunities-early-adopters/opportunities-early-adopters-page.component.html:65]: I mean `class` would work but why not just style `h4` directly?

File: src/app/welcome-page/welcome-page.component.html [hunk 2]
```diff
@@ -50,7 +50,7 @@ <h1>
     <img class="section-img" src="/info_section_1_illustration.svg" />
 
     <div class="box">
-      <h2><span> Document the evidence,</span> unmask the trolls</h2>
+      <h2>Document the evidence, unmask the trolls</h2>
```
Comment (kilemensi) [inline src/app/welcome-page/welcome-page.component.html:53]: Damn, we're moving away from 2 level-heading style????

**Review by kilemensi — COMMENTED** [review]
👍🏽

File: src/app/welcome-page/welcome-page.component.html (hunk 2 above)
Comment (kelvinkipruto) [reply src/app/welcome-page/welcome-page.component.html:53]: Yes. They want it all uniform.

File: src/app/opportunities-early-adopters/opportunities-early-adopters-page.component.html (hunk 1 above)
Comment (kelvinkipruto) [reply src/app/opportunities-early-adopters/opportunities-early-adopters-page.component.html:65]: MMh, I should have. Much simpler.

File: src/app/opportunities-early-adopters/opportunities-early-adopters-page.component.scss [hunk 3]
```diff
@@ -43,8 +43,21 @@
...
+    font-size: 22px;
+    line-height: math.div(27, 22);
+  }
+
+  h4 {
+    font-size: 24px;
+    line-height: 1.125;
+    font-size: 22px;
+    font-weight: 700;
+    line-height: math.div(27, 22);
```
Comment (kilemensi) [inline src/app/opportunities-early-adopters/opportunities-early-adopters-page.component.scss:55]: 2 font sizes and line heights?

File: src/app/opportunities-early-adopters/opportunities-early-adopters-page.component.scss [hunk 4]
```diff
@@ -43,8 +43,19 @@
...
-    font-size: 16px;
-    line-height: math.div(27, 16);
+    font-size: 22px;
+    line-height: math.div(27, 22);
+  }
+
+  h4 {
+    font-size: 24px;
+    font-weight: 700;
+    line-height: math.div(27, 22);
```
Comment (kilemensi) [inline src/app/opportunities-early-adopters/opportunities-early-adopters-page.component.scss:53]:
```suggestion
    line-height: math.div(27, 24);
```

Should be divided by `font-size`.

**Review by kilemensi — APPROVED** [review]
🚀

**Pull request merged**